### PR TITLE
Add 'claim' command

### PR DESCRIPTION
**Pull request by morfj**

### CODE REVIEW

File: src/commands/claim-nii.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            spinner.succeed('4/6 - Transfer approval confirmed');
+
+            spinner.start('5/6 - Registering nahmii deposit');
+            const pendingCompleteTx = await wallet.completeTokenDeposit(niiBalance, 'NII', options);
+            spinner.succeed('5/6 - nahmii deposit registered');
+
+            spinner.start('6/6 - Confirming nahmii deposit');
+            const completeReceipt = await provider.getTransactionConfirmation(pendingCompleteTx.hash);
+            spinner.succeed(`6/6 - nahmii deposit of ${niiBalance} NII confirmed`);
+
```
Comment (jonbern) [inline src/commands/claim-nii.js]: What happens if one these steps fail due to a transient error condition? Is it possible to re-run the `claim` command and eventually get a successful run?

File: src/commands/claim-nii.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+};
+
+let validateGasLimitIsPositiveInteger = function(gas) {
+    const gasLimit = parseInt(gas);
+    if (gasLimit <= 0)
+        throw new Error('Gas limit must be a number higher than 0');
+    return gasLimit;
+};
+
+let validatePeriodIsPositiveInteger = function(period) {
```
Comment (jonbern) [inline src/commands/claim-nii.js]: I think `validatePeriod` would be a more accurate name for this function. 

Also, any particular reason why you're not using `const`?

**Review by jonbern — DISMISSED** [review]

File: src/commands/claim-nii.js (hunk 1 above)
Comment (morfj) [reply src/commands/claim-nii.js]: The process can be divided into 2 main parts, a) claiming funds and b) depositing said funds to nahmii.
a) is implemented by step 1-2. If they fail, no biggie, you can then re-run the command as-is. 
b) is implemented by step 4-6 and If one of them fail, then you have your funds available on-chain but have to "rerun" by doing a regular "deposit" command. 

Nothing is actually deposited until step 6 has been mined.

File: src/commands/claim-nii.js (hunk 2 above)
Comment (morfj) [reply src/commands/claim-nii.js]: Copy paste syndrome.. changed to regular named functions.

**Review by jonbern — APPROVED** [review]

**Review by katat — APPROVED** [review]

**Pull request merged**